app_dart/pubspec.yaml

```diff
@@ -36,6 +36,7 @@ dev_dependencies:
   fake_async: ^1.2.0
   json_serializable: ^6.0.0
   mockito: ^5.0.14
+  pedantic: ^1.11.1
```

Comment:
why are we adding this?
https://pub.dev/packages/pedantic

Reply:
We use this for `unawaited` in `dev/deploy.dart`. This was a transitive dependency before, and was removed completely with the original update